src/core/reducers/addons.js

```diff
@@ -242,13 +241,21 @@ export function createInternalAddon(
     themeData: createInternalThemeData(apiAddon),
   };
 
-  const currentVersion = apiAddon.current_version;
-
   if (
     currentVersion &&
     currentVersion.files &&
     currentVersion.files.length > 0
   ) {
+    const { files } = currentVersion;
+    const fileSizeInBytes = files
+      .map((file) => file.size)
+      .reduce((accumulator, size) => accumulator + size);
+    const fileSize =
+      fileSizeInBytes <= 10 ** 5
+        ? `${(fileSizeInBytes / 10 ** 3).toFixed(1)} KiB`
+        : `${(fileSizeInBytes / 10 ** 6).toFixed(1)} MiB`;
```

Comment:
We have a `formatFilesize()` helper in `src/core/i18n.utils.js` to translate a file size in bytes into a localized user-friendly format.

In addition, I think we should avoid adding two many props on the `addons` object. We could probably compute the size of the version in `AddonsMoreInfo`?

Reply:
Thanks for reviewing. 
Sure, I'll send in another patch with the aforementioned changes soon.